Queue/Backend/Redis.php

```diff
@@ -233,7 +233,7 @@ private function connectIfNeeded()
     protected function connect()
     {
         $this->redis = new \Redis();
-        $success = $this->redis->connect($this->host, $this->port, $this->timeout, null, 100);
+        $success = $this->redis->pconnect($this->host, $this->port, $this->timeout, null, 100);
```

Comment:
Is this change needed? Doesn't this open a persistent connection? It seems to be a feature of phpredis? but I am actually not sure whether we are still using PHPredis. I think we have replaced it with credis. Couldn't find a command pconnect